configs/sample_free/init_prior.json

```diff
@@ -0,0 +1,2 @@
+{"FasterRCNN_ResNet50_FPN_COCO": 0.0001584489073138684,
```

Comment:
It seems that we do not need a json file to store these values, making it a hyperparameter to the head should work.